### DISCUSSION TITLE
[Question] where to add user commands

### DISCUSSION THREAD

**Discussion by rs7q5** [James-Yu/LaTeX-Workshop#3807] [Q&A] [answered]
In TexStudio I have a user command called `Make Nomenclature` and the command is `makeindex -s nomencl.ist -t %.nlg -o %.nls %.nlo` 

Do I stick that in the recipes/settings file to use that command or where would this go in the settings for this extension?

**Comment by James-Yu** [comment, marked as answer]
Please check wiki on compiling a document.

**Reply by rs7q5** [reply]
I was checking as I wasn't sure if it needed to be a tool and a recipe. Seems that was in fact the case.
